### PR TITLE
Added initial city record exercise solution

### PR DESCRIPTION
**Pull request by antoniaat**

### CODE REVIEW

File: 03-objects-and-composition/lab/01-city-record.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function createRecord(name, population, treasury) {
+  const city = {};
+  city.name = name;
+  city.population = population;
+  city.treasury = treasury;
+  
+  return city;
```
Comment (antoniaat) [inline 03-objects-and-composition/lab/01-city-record.js:7]: Maybe you can directly return the object using syntax like this (so to simplify the function):

 ```
function createRecord(name, population, treasury) {
  return {
    name,
    population,
    treasury,
  };
}

```

**Review by georgi-georgiev95 — APPROVED** [review]
Pure code!

File: 03-objects-and-composition/lab/01-city-record.js (hunk 1 above)
Comment (StanislavMakulev) [reply 03-objects-and-composition/lab/01-city-record.js:7]: Good option!

**Review by StanislavMakulev — APPROVED** [review]
Good option!

**Pull request merged**